pkg/controller/model/constants.go

```diff
@@ -26,4 +26,5 @@ const (
 	GrafanaAdminPasswordEnvVar          = "GF_SECURITY_ADMIN_PASSWORD"
 	GrafanaHttpPort                 int = 3000
 	GrafanaHttpPortName                 = "grafana"
+	GrafanaDBPasswordVar                = "GF_DATABASE_PASSWORD"
```

Comment:
GrafanaDBPasswordEnv?

Reply:
Better is `GrafanaDBPasswordEnvVar` for consistency